src/app/core/user/user.model.js

```diff
@@ -304,6 +375,10 @@ UserSchema.statics.fullCopy = function(user) {
 		if (_.has(toReturn, 'password')) {
 			delete toReturn.password;
 		}
+		if (_.has(toReturn, 'salt')) {
```

Comment:
I assume we didn't actually want to return the salt property, even if it's behind an admin role auth.

Reply:
Good catch.